### DISCUSSION TITLE
[Bug] There is an error in the collection time

### DISCUSSION THREAD

**Discussion by 284885166** [apache/skywalking#12723] [Q&A]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar issues.

### Apache SkyWalking Component

OAP server (apache/skywalking)

### What happened

Why is there a three second interval

### What you expected to happen

trace spans start time

### How to reproduce

java

### Are you willing to submit a pull request to fix on your own?

- [ ] Yes I am willing to submit a pull request on my own!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by 284885166** [comment]
![image](https://github.com/user-attachments/assets/3dff8255-f2ff-4692-9f3a-84314364d421)

**Comment by wu-sheng** [comment]
I am not following your question. 0ms means less than 1ms. Nothing special.

**Reply by 284885166** [reply]
两次时间间隔了三秒钟，是什么原因造成的？是采集丢失了吗？

**Reply by wu-sheng** [reply]
Please put English on GitHub, if you need to discuss in Chinese, please use slack CN channel.

I can't tell the reason, but maybe some codes in between, or jedis internal blocks. If you want to know that, learn how to use profiling.

**Reply by 284885166** [reply]
Thank you, I have found the reason.
